.env.example

```
@@ -82,7 +97,7 @@ EMAIL_SENDER=
 # Database Configuration
 DB_HOST=localhost # May need to be changed to `DB_HOST=db` if using devcontainer
 DB_PORT=5432
-POSTGRES_PASSWORD=postgres
+POSTGRES_PASSWORD=postgres # pipelock:ignore
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)